02-frequency_filter.py

```diff
@@ -92,8 +92,7 @@ def run_filter(subject, run=None, session=None):
         raw.plot(n_channels=50, butterfly=True)
```

Comment:
Is `n_jobs = 1` necessarily when `config.plot=True`?

Reply:
Very good question, and it would apply to the entire pipeline!